### PR TITLE
workflows: More re-skinning invocation page

### PR DESCRIPTION
**Pull request by bduffany**
* Trim invocation details card
* Remove non-applicable cards from "All" view in non-dense mode
* Remove non-applicable items from overview header (both dense and non-dense)
* In dense mode, update the status text to "1 action" instead of "0 targets".

**Before:**

image

**After:**

image

---

**Version bump**: Patch

### CODE REVIEW

File: app/invocation/dense/dense_invocation_overview.tsx [hunk 1]
```diff
@@ -46,11 +48,26 @@ export default class DenseInvocationOverviewComponent extends React.Component {
...
-            {this.props.model.getStartDate()} at {this.props.model.getStartTime()} for{" "}
-            <span title={this.props.model.getDurationSeconds()}>{this.props.model.getTiming()}</span>
+        <div
+          className={`dense-invocation-status-bar ${this.props.model.getStatusClass()} ${
+            !isBazelInvocation && "dense-ci-run-status-bar"
+          }`}>
+          <div className="dense-invocation-status-bar-left">
+            {!isBazelInvocation && <div className="ci-run-status-icon">{this.props.model.getStatusIcon()}</div>}
+            <div>
+              {isBazelInvocation ? (
```
Comment (siggisim) [inline app/invocation/dense/dense_invocation_overview.tsx:58]: I'm fine with showing / hiding things based on whether or not we're dealing with a CI run, but once we start making the UI look different - we might as well create separate components.

Can we either:
a) Include the success / failure in the status bar for all invocations 
b) Just hide the `dense-invocation-overview-grid-chunk`s below that we want to hide
c) Hide all of the `dense-invocation-overview-grid-chunk` that have zero values

File: app/invocation/invocation_model.tsx [hunk 2]
```diff
@@ -261,8 +263,16 @@ export default class InvocationModel {
...
+    return this.getRole() !== CI_RUNNER_ROLE;
+  }
+
   getTool() {
-    return `bazel v${this.started?.buildToolVersion} ` + this.started?.command || "build";
+    if (this.isBazelInvocation()) {
+      return `bazel v${this.started?.buildToolVersion} ` + this.started?.command || "build";
+    }
+
+    return "BuildBuddy CI runner";
```
Comment (siggisim) [inline app/invocation/invocation_model.tsx:275]: I do think it's a little confusing that in some places in the UI (user facing) these are called workflows, some places they're called actions, and now we're also calling them "CI runs".

File: app/invocation/invocation_overview.tsx [hunk 3]
```diff
@@ -71,7 +72,7 @@ export default class InvocationOverviewComponent extends React.Component {
             <span>Invocation {this.props.invocationId}</span>
           </div>
           <div className="invocation-top-right-buttons">
-            <InvocationCompareButton invocationId={this.props.invocationId} />
+            {isBazelInvocation && <InvocationCompareButton invocationId={this.props.invocationId} />}
```
Comment (siggisim) [inline app/invocation/invocation_overview.tsx:75]: Don't comparisons work on raw events - shouldn't this just work?

File: app/invocation/invocation_overview.tsx [hunk 4]
```diff
@@ -4,7 +4,7 @@ import format from "../format/format";
 import router from "../router/router";
 import InvocationCompareButton from "./invocation_compare_button";
 import InvocationMenuComponent from "./invocation_menu";
-import InvocationModel from "./invocation_model";
+import InvocationModel, { CI_RUNNER_ROLE } from "./invocation_model";
```
Comment (siggisim) [inline app/invocation/invocation_overview.tsx:7]: Is this CI_RUNNER_ROLE import needed?

**Review by siggisim — CHANGES_REQUESTED** [review]

File: app/invocation/dense/dense_invocation_overview.tsx (hunk 1 above)
Comment (bduffany) [reply app/invocation/dense/dense_invocation_overview.tsx:58]: Went with option (a) - WDYT?

Comment (bduffany) [reply app/invocation/dense/dense_invocation_overview.tsx:56]: Changed to option (b) per offline discussion

**Review by siggisim — APPROVED** [review]

**Pull request merged**